images/macos/software-report/SoftwareReport.Xcode.psm1

```diff
@@ -220,6 +220,40 @@ function Build-XcodeSimulatorsTable {
     }
 }
 
+function Build-XcodeCommandLineToolsSection {
+    $PKG_PATH = "/Library/Developer/CommandLineTools"
+
+    $clang_version = (&"${PKG_PATH}/usr/bin/clang" --version).Split(' ')[3]
+
+    $clang_version_parts = $clang_version.Split('.')
+    $clang_version_no = ($clang_version_parts[0] -as [int]) * 10 + ($clang_version_parts[1] -as [int])
+
+    $xcode_version = switch ($clang_version_no) {
```

Comment:
Is there any way to avoid maintenance of this list? Looks like we will have to update it manually in future.
I would even say that this list is already outdated for now.

I think we can retrieve CLT version via pkgutil tool: `pkgutil --pkg-info com.apple.pkg.CLTools_Executables`
```
pkgutil --pkg-info com.apple.pkg.CLTools_Executables | grep "version" | cut -d' ' -f 2 | cut -d'.' -f 1,2
```
should work without maitenance